lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -1335,8 +1350,12 @@ private Task requestChanges(String taskId, boolean force)
     return task;
   }
 
-  private static void claimActionsOnTask(TaskSummaryImpl task, String userId, Instant now) {
+  private static void claimActionsOnTask(TaskSummaryImpl task, String userId, String userLongName,
+                                         Instant now, Boolean additionalUserInfo) {
     task.setOwner(userId);
+    if (additionalUserInfo) {
```

Comment:
is this really necessary?

Reply:
Should I set ownerLongName to the value of userLongName in any case? It will be just set to null if additionalUserInfo is not set.

Reply:
yes I would prefer that, since it would be null anyways if it is not set at all.